pkg/service/serve/serve.go

```diff
@@ -22,6 +22,7 @@ import (
 	"go.keploy.io/server/pkg/service/serve/graph"
 	"go.keploy.io/server/pkg/service/test"
 	"go.uber.org/zap"
+	sentry "github.com/getsentry/sentry-go"
```

Comment:
Can we add an event which will tell us that users are using Junit integration..?

Reply:
I talked to the team about that, it was finalised that we will keep it as test run as of now.

Reply:
okay